frontend/src/container/CreateAlertChannels/index.tsx

```diff
@@ -21,26 +21,16 @@ function CreateAlertChannels({
 	const [selectedConfig, setSelectedConfig] = useState<
 		Partial<SlackChannel & WebhookChannel>
 	>({
-		text: ` {{ range .Alerts -}}
-     *Alert:* {{ .Annotations.title }}{{ if .Labels.severity }} - {{ .Labels.severity }}{{ end }}
+		text: `{{ range .Alerts -}}
+     *Alert:* {{ .Labels.alertname }}{{ if .Labels.severity }} - {{ .Labels.severity }}{{ end }}
 
+     *Summary:* {{ .Annotations.summary }}
      *Description:* {{ .Annotations.description }}
 
      *Details:*
        {{ range .Labels.SortedPairs }} • *{{ .Name }}:* {{ .Value }}
        {{ end }}
      {{ end }}`,
-		title: `[{{ .Status | toUpper }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}] {{ .CommonLabels.alertname }} for {{ .CommonLabels.job }}
```

Comment:
@palash-signoz Is this removal intentional?

Reply:
My bad for not clearly mentioning it. 😅️

The text I shared was for the placeholder for the description. Here, `text` key is used for that.
We use the `title` key for the placeholder for the title input box. We will need it back.

Reply:
NP _(with Axolo)_